src/main/java/edu/hm/hafner/analysis/parser/CodeGeneratorParser.java

```diff
@@ -0,0 +1,77 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.io.UncheckedIOException;
+import java.util.Iterator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.ReaderFactory;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+/**
+ * A parser for Simulink Code Generator tool log files.
+ *
+ * @author Eva Habeeb
+ */
+public class CodeGeneratorParser extends IssueParser {
```

Comment:
Can't you use `LookaheadParser` as base class? Most of the code is duplicated here

Reply:
Fixed.